config/packages/rate_limiter.yaml

```diff
@@ -116,3 +116,7 @@ framework:
       policy: 'fixed_window'
       limit: 4
       interval: '1 day'
+    ap_update_actor:
+      policy: 'sliding_window'
+      limit: 1
+      interval: '5 minutes'
```

Comment:
Why not use 60 minutes here, too?

Reply:
my intention here is to limit a burst of multiple update messages being dispatched in quick succession (from my observation, when its time to update it tends to go dispatching like six update actor at once and then basically none after update goes through), while still allow for faster retry if the dispatched message earlier failed to update the actor, e.g. from intermittent connectivity issue to remote instance

Reply:
And the lock only blocks the processing while one is currently running right? If yes I just misunderstood it before